ts/components/ui/utils/baseComponents/LogoPaymentOrDefaultIcon.tsx

```diff
@@ -1,8 +1,13 @@
 import * as O from "fp-ts/lib/Option";
```

Comment:
We already have `getCardLogoComponent` that does the same thing in `ts/features/idpay/common/components/CardLogo.tsx`
We could extend that and move to a root utils folder. What do you think?

Reply:
As suggested by @mastro993 the `getCardLogoComponent` already do the "hard work", maybe you could handle only the normalization of the brand string.

Anyway the implemented `LogoPaymentOrDefaultIcon` seems to have a unexpected behavior since when I go into a credit card detail with a "visa" brand, in its detail shows the fallback icon as showed in the following video:


https://github.com/pagopa/io-app/assets/34343582/035d6b67-6345-450a-9f10-d47ff123f605

Reply:
@mastro993  true, i was thinking of removing the IDPay util and moving it all to the global util; what do you think?
@Hantex9  that is in fact expected behaviour since visElectron is not a mapped brand in `IOLogoPaymentType`, after a short discussion the answer is:
- it is a circuit which is getting dismissed
- we should check if the actual backend -- not the DevServer -- returns a different brand for visaElectron compared to a normal visa, in which case we should add the icon to the type

also, this is the exact reason why `LogoPaymentOrDefaultIcon` exists! that is the kind of edge case it is trying to catch

Reply:
Yes, we could move `ts/features/idpay/common/components/CardLogo.tsx` to `ts/components/CardLogo.tsx`. It is more like a component other than an utility.
We should also extend the `getCardLogoComponent` function to accept an icon color.

Reply:
> @mastro993 true, i was thinking of removing the IDPay util and moving it all to the global util; what do you think? @Hantex9 that is in fact expected behaviour since visElectron is not a mapped brand in `IOLogoPaymentType`, after a short discussion the answer is:
> 
> * it is a circuit which is getting dismissed
> * we should check if the actual backend -- not the DevServer -- returns a different brand for visaElectron compared to a normal visa, in which case we should add the icon to the type
> 
> also, this is the exact reason why `LogoPaymentOrDefaultIcon` exists! that is the kind of edge case it is trying to catch

`getCardLogoComponent` does the same exact thing and it is already well tested :)